README.md

```diff
@@ -554,6 +554,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [consul-api](https://github.com/Ecwid/consul-api) - Client for the [Consul](https://www.consul.io/) API a distributed, highly available and datacenter-aware registry/discovery service.
 * [Eureka](https://github.com/Netflix/eureka) - REST based service registry for resilient load balancing and failover.
 * [Lagom](https://www.lightbend.com/lagom) - Framework for creating microservice-based systems.
+* [Light Java](https://github.com/networknt/light-java) - A fast and configurable REST microservices framework for docker containers on the cloud.
+* [Ja-micro](https://github.com/Sixt/ja-micro) - Lightweight framework for building java microservices based off [Go Micro](https://github.com/micro)
```

Comment:
See contribution rules:

1. If an entry has a similar scope as other entries in the same category, the description must state the unique features that distinguishes it from the other entries.